_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -3,7 +3,7 @@ layout: post
 title: "Supercharged SEO with Drupal 8"
 date: 2016-09-14
 author: Oksana Cyrwus
-tags: drupal drupal8 marketing
+tags: drupal drupal8 drupal-planet marketing 
```

Comment:
@oksana-c since we're adding the drupal planet 💧 🌍 tag in hopes to get it on Drupal planet, we can also add some front matter to feature your awesome image ( a must) as well as give a concise summary for Drupal planet. Please work with @AnneTee to get this done the right way. http://127.0.0.1:4000/2016/02/24/drupal-6-part-4.html is a good example to follow. After that, we'll be good to go!